src/ui/components/Timeline/Marker.tsx

```diff
@@ -33,73 +32,67 @@ type MarkerProps = PropsFromRedux & {
   pauseId?: PauseId;
 };
 
-class Marker extends React.Component<MarkerProps> {
-  shouldComponentUpdate(nextProps: Readonly<MarkerProps>) {
-    const highlightChanged = this.props.isPrimaryHighlighted !== nextProps.isPrimaryHighlighted;
-
-    return (
-      highlightChanged ||
-      this.props.time !== nextProps.time ||
-      this.props.currentTime !== nextProps.currentTime ||
-      this.props.overlayWidth !== nextProps.overlayWidth ||
-      this.props.zoomRegion !== nextProps.zoomRegion
-    );
-  }
 
-  onClick: MouseEventHandler = e => {
-    const { seek, point, time, pauseId } = this.props;
+const Marker = (props : MarkerProps) => {
```

Comment:
:hand: We normally destructure all the props inside the component function declaration - that way we don't have to do it again later. Can you do that here?